packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -136,7 +136,11 @@ class PatchIosCommand extends ShorebirdCommand
       releaseVersion: releaseVersion,
     );
 
-    if (release.platformStatuses[ReleasePlatform.ios] == ReleaseStatus.draft) {
+    if (release.platformStatuses[ReleasePlatform.ios] == null) {
+      logger.err('No iOS release found for $releaseVersion.');
```

Comment:
I suspect you'll end up abstracting this log.  Every platform is going to want it, just with "iOS" substituted.

Reply:
Yeah, I think we probably will want to consolidate our release and patch commands before too long. Lots of duplicated logic (and test logic) between them.